clients/cobol-lsp-vscode-extension/src/services/ControlFlowService.ts

```diff
@@ -71,36 +81,64 @@ interface AnalysisServiceDelegate {
 }
 
 class AnalysisTask {
-  private worker: Worker = new Worker(
-    join(__dirname, "../src/services/Worker.js"),
-  );
+  private worker: Worker = new Worker(join(__dirname, "./Worker.js"));
```

Comment:
Is the `__dirname` actually required? Does relative path work?

Reply:
"Error starting Control Flow Analysis: Error: Cannot find module '/Worker.js'"